packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/views/ViewManagerDefinitionBuilder.kt

```diff
@@ -37,4 +41,11 @@ class ViewManagerDefinitionBuilder {
       body
     )
   }
+
+  /**
+   * Defines prop names that should be treated as callbacks.
+   */
+  fun callbacks(vararg callbacks: String) {
```

Comment:
```suggestion
  fun events(vararg callbacks: String) {
```
@tsapeta, I saw that you called it `events` on iOS. Should I rename it?

Reply:
Yeah, I think it makes sense to call them events, as React Native does. I also just reused the component that we have in the module scope 😉 
Imho the name and type "Callback" can also be used in the future for function arguments that are functions on the JS side 🤔